_pages/team.html

```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
             <h5>Keenan Hom</h5>
             <p>Keenan is a mechanic.</p>
         </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Connor Floyd</h5>
+            <!---TODO: add details---->
+            <p>Connor is a programmer. He has helped to edit our online guides and to code the website.</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Connor Mills</h5>
+            <!---TODO: add details---->
+            <p>Bumpers. Mechanic</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Fernanda Molina</h5>
+            <!---TODO: add details---->
+            <p>Programming</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Blaine Morse</h5>
+            <!---TODO: add details---->
+            <p>Programming</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/nathaniel-horne.png">
+            <h5>Johnny Assel</h5>
+            <!---TODO: add details---->
+            <p>Nathaniel is a programmer. He is the de facto editor of the website itself.</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Johnny Assel</h5>
+            <!---TODO: add details---->
+            <p>Johnny works with strategy and scoutting. His proficiency with scoutting data has been invaluable to the team.</p>
```

Comment:
"Scouting."

Also, it hasn't really been helpful yet... not that it won't be in the future.